lib/config.js

```diff
@@ -25,3 +25,8 @@ cfg.routes = {
   registration: '/authn/token/client/registration',
   recovery: '/authn/token/recovery'
 };
+
+cfg.fakeTokenOptions = {
```

Comment:
Would be good to have one-liner docs on what these are for.  And mention what the `hmacKey` value should be.  Is it supposed to be a long random string?  Is it a critical secret?